0011-command-execution-protocol/README.md

```diff
@@ -0,0 +1,170 @@
+---
+name: Command Execution Protocol
+shortname: 11/CEP
+status: raw
+editor: Alexey Shmalko <ashmalko@kaaiot.io>
+---
+
+<!-- toc -->
+
+# Introduction
+Command Execution Protocol is an endpoint-aware 1/KP extension. It is designed to allow command execution on endpoints.
```

Comment:
Ok, it's friendlier to *my* eyes ) Because it helps me follow logical flow in a paragraph, make edits and text manipulations easier. However, when I edit on a small laptop screen and use text wrap, the sentence-per-line method stops making any difference.

Well, it's subjective, and I personally have no problem with relaxing this rule and letting everyone write as they want.